services/udp/service.go

```diff
@@ -106,7 +106,7 @@ func (s *Service) writePoints() {
 		case batch := <-s.batcher.Out():
 			if err := s.PointsWriter.WritePoints(&cluster.WritePointsRequest{
 				Database:         s.config.Database,
-				RetentionPolicy:  "",
+				RetentionPolicy:  s.config.RetentionPolicy,
```

Comment:
Yes, because `""` means use default. This is what happens now, but you have no choice but to accept `""` in the code.